tabular/src/autogluon/tabular/models/realmlp/realmlp_model.py

```diff
@@ -274,9 +274,20 @@ def _get_default_stopping_metric(self):
     def _get_default_resources(self) -> tuple[int, int]:
         # Use only physical cores for better performance based on benchmarks
         num_cpus = ResourceManager.get_cpu_count(only_physical_cores=True)
+        
         # Only request GPU if CUDA is available (RealMLP doesn't support MPS)
-        import torch
-        num_gpus = 1 if torch.cuda.is_available() else 0
+        try:
+            import torch
+            num_gpus = 1 if torch.cuda.is_available() else 0
```

Comment:
can we instead add a flag to the `ResourceManager.get_gpu_count_torch` to ignore the MPS logic? This seems like it would be cleaner.